pkg/tfbridge/info.go

```diff
@@ -463,17 +466,18 @@ func (m *MarshallableProvider) Unmarshal() shim.Provider {
 
 // MarshallableSchemaInfo is the JSON-marshallable form of a Pulumi SchemaInfo value.
 type MarshallableSchemaInfo struct {
-	Name        string                             `json:"name,omitempty"`
-	CSharpName  string                             `json:"csharpName,omitempty"`
-	Type        tokens.Type                        `json:"typeomitempty"`
-	AltTypes    []tokens.Type                      `json:"altTypes,omitempty"`
-	Elem        *MarshallableSchemaInfo            `json:"element,omitempty"`
-	Fields      map[string]*MarshallableSchemaInfo `json:"fields,omitempty"`
-	Asset       *AssetTranslation                  `json:"asset,omitempty"`
-	Default     *MarshallableDefaultInfo           `json:"default,omitempty"`
-	MaxItemsOne *bool                              `json:"maxItemsOne,omitempty"`
-	Deprecated  string                             `json:"deprecated,omitempty"`
-	ForceNew    *bool                              `json:"forceNew,omitempty"`
+	Name         string                             `json:"name,omitempty"`
+	CSharpName   string                             `json:"csharpName,omitempty"`
+	Type         tokens.Type                        `json:"typeomitempty"`
+	AltTypes     []tokens.Type                      `json:"altTypes,omitempty"`
+	Elem         *MarshallableSchemaInfo            `json:"element,omitempty"`
+	Fields       map[string]*MarshallableSchemaInfo `json:"fields,omitempty"`
+	Asset        *AssetTranslation                  `json:"asset,omitempty"`
+	Default      *MarshallableDefaultInfo           `json:"default,omitempty"`
+	MaxItemsOne  *bool                              `json:"maxItemsOne,omitempty"`
+	Deprecated   string                             `json:"deprecated,omitempty"`
+	ForceNew     *bool                              `json:"forceNew,omitempty"`
+	MarkAsSecret *bool                              `json:"markAsSecret,omitempty"`
```

Comment:
Total nit - but any reason to include "MarkAs" in the name?  No other properties have this, even though they all in principle "mark as".  Should this just be `Secret`?